modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -122,20 +122,7 @@
 /datum/job/science_guard //See orderly
 
 /datum/job/scientist
-	alt_titles = list(
```

Comment:
why'd you change this part to be a worse formatted list

Reply:
because none of the others are like that

Reply:
after this is merged, i'll probably get around to making them all like that, but at the minute, none of the others are like that and it's, obviously, inconsistent

Reply:
You should newline after each entry on a list like this. This isn't formatting for formatting's sake, it's for making git blame easier to use.